static/js/client.js

```diff
@@ -155,16 +156,17 @@ function buildTAEntry(entry) {
     } else if (entry.status == 1) {
         elt.find(".helping-text").html(`${entry.ta_full_name} is helping ${xHtml}`);
     } else if (!ta_helping_id) {
+        elt.find(".remove-button").removeClass("hide");
         if (ta_id) {
-            elt.find(".remove-button").removeClass("hide");
-            if (!entry.blocked) {
-                elt.find(".help-button").removeClass("hide");
+            elt.find(".help-button").removeClass("hide");
+            if (!entry.update_requested) {
                 elt.find(".fix-question-button").removeClass("hide");
             } else {
+                elt.find(".help-button")
+                    .removeClass("waves-light btn blue")
+                    .addClass("waves btn-flat grey lighten-3 grey-text text-darken-2");
                 elt.find(".helping-text").text("Student is updating question");
             }
```

Comment:
This would be clearer if the logic was flipped:
```
if (entry.update_requested) {
  // code if update is requested 
} else {
  // code if update isn't requested
}
```